src/lib/y2storage/filesystems/blk_filesystem.rb

```diff
@@ -118,6 +118,18 @@ def plain_blk_devices
         blk_devices.map(&:plain_device)
       end
 
+      def name
+        textdomain "storage"
+
+        format(
+          # TRANSLATORS: %{fs_type} is replaced by the filesystem type (e.g., BtrFS), and %{device_name}
```

Comment:
yes, but this still does not tell the translator the *context* of the message that may be useful for the translation